app/lib/features/cross_signing/pages/request_ready.dart

```diff
@@ -0,0 +1,102 @@
+import 'package:acter/common/themes/app_theme.dart';
+import 'package:atlas_icons/atlas_icons.dart';
+import 'package:flutter/material.dart';
+
+class RequestReadyPage extends StatelessWidget {
+  final bool isVerifier;
+  final Function(BuildContext) onCancel;
+  final Function(BuildContext) onAccept;
+
+  const RequestReadyPage({
+    super.key,
+    required this.isVerifier,
+    required this.onCancel,
+    required this.onAccept,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      decoration: BoxDecoration(
+        borderRadius: BorderRadius.circular(15),
+      ),
+      child: Column(
+        mainAxisAlignment: MainAxisAlignment.start,
+        mainAxisSize: MainAxisSize.min,
+        children: [
+          Padding(
+            padding: const EdgeInsets.symmetric(vertical: 5),
+            child: buildTitleBar(context),
+          ),
+          const Spacer(),
+          const Padding(
+            padding: EdgeInsets.symmetric(horizontal: 15),
+            child: Text(
+              'Scan the code with your other device or switch and scan with this device',
+            ),
+          ),
+          const Spacer(),
+          const Center(
+            child: Padding(
+              padding: EdgeInsets.all(25),
+              child: SizedBox(
+                width: 50,
+                height: 50,
+                child: CircularProgressIndicator(),
+              ),
+            ),
+          ),
+          const Spacer(),
+          // TextButton(
```

Comment:
This leaves commented. We can remove it!

Reply:
This part was left for future implement of qrcode scan.